cypress/e2e/models/migration/archetypes/archetype.ts

```diff
@@ -192,4 +192,46 @@ export class Archetype {
             }
         }
     }
+
+    duplicate(
+        name?: string,
+        criteriaTags?: string[],
+        archetypeTags?: string[],
+        description?: string,
+        stakeholders?: Stakeholders[],
+        stakeholderGroups?: Stakeholdergroups[],
+        comments?: string,
+        cancel = false
+    ) {
+        Archetype.open();
```

Comment:
@midays You can just call the create method inside this method .

Reply:
it's not possible, when I tried it, it requried refactoring in the create method, I was able to do some of the refactoring required but, I got stuck in a situation where I can't just click duplicate without having to edit any details. In the create method it treats the passed arguments as new ones and it fills each of them, to handle what I need I need to pass an extra bool parameter for each existing parameter to decide whether to add or not, so it's just more efficient to do it separatly in the duplicate method